dev/environment

```diff
@@ -64,6 +64,9 @@ TWOFACTORMANDATE_AVAILABLE=true
 TWOFACTORMANDATE_ENABLED=true
 OIDC_AUDIENCE=pypi
 
+TUF_RSTUF_API_URL="http://rstuf-api"
+TUF_ENABLED=true
```

Comment:
Btw. in RubyGems.org we use `RSTUF_API_URL` and we enable it just by presence. Not sure if worth to keep it similar.

https://github.com/rubygems/rubygems.org/blob/ad1269b77e3ac83edac6a8f399125ba9fec0280d/config/initializers/rstuf.rb#L3C9-L7